src/NServiceBus.Msmq.AcceptanceTests/EndpointCustomizationConfigurationExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace NServiceBus.Transport.Msmq.AcceptanceTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using AcceptanceTesting.Support;
+    using Hosting.Helpers;
+    using ObjectBuilder;
+
+    public static class EndpointCustomizationConfigurationExtensions
```

Comment:
this is needed by `DefaultServer` and is shipped with the acceptance test sources. We can just remove those classes again once the transition is done.